incubator/hnc/pkg/controllers/hierarchy_controller_test.go

```diff
@@ -190,38 +186,38 @@ var _ = Describe("Hierarchy", func() {
 		// Verify that bar has a tree label related to foo
 		Eventually(func() bool {
 			barNS := getNamespace(ctx, barName)
-			_, ok := barNS.GetLabels()[fooName+".tree."+metaGroup+"/depth"]
+			_, ok := barNS.GetLabels()[fooName+".tree."+api.MetaGroup+"/depth"]
```

Comment:
At some point (not in this PR), maybe consider turning ".tree.x-k8s.io/depth" into a const, even if it's just in this package.

Reply:
Sounds good!